Exercises/numbers/numbers.js

```diff
@@ -1,3 +1,65 @@
-function extractNumbers (text) {
-  return null;
-}
\ No newline at end of file
+// # Numbers
+// 1. Assign by hand same value to 3 variables in binary, octal and hexadecimal format.
+let binary = '1111';
+let octal = '17';
+let hexadecimal = '0xf';
+
+binary = parseInt(binary, 2);
+octal = parseInt(octal, 8);
+hexadecimal = parseInt(hexadecimal, 16);
+
+console.log(binary == octal, octal == hexadecimal, hexadecimal == binary);
+
+// 2. Implement function `extractNumbers` which will leave only numbers in passed string.
+function extractNumbers(str) {
+  let resultString = '';
+  for (let char in str) {
```

Comment:
good idea! but we can improve, just iterate through string and check is current symbol number is yes - add to` resultString`